.github/workflows/ci.yml

```diff
@@ -7,24 +7,30 @@ on:
   # Every Thursday at 12:20pm UTC
   schedule:
     - cron: '20 12 * * 4'
-
 jobs:
   ci:
     name: CI
     # Only run cron on the silverstripe account
     if: (github.event_name == 'schedule' && github.repository_owner == 'silverstripe') || (github.event_name != 'schedule')
-    uses: silverstripe/gha-ci/.github/workflows/ci.yml@v1
+    uses: "maxime-rainville/gha-ci/.github/workflows/ci.yml@1"
```

Comment:
Do we need to run the fork?

Reply:
That's just a temporary thing until we get the CI set up sorted.

`silverstripe/gha-ci` would want to delete all the test under `vendor/silverstripe/framework/tests/php`. The module itself has almost no test coverage ... it relies on the tests from framework under the assumption that testing the ORM will test the PostgreSQL DB adapter.

Reply:
Still needs to be set back to silverstripe